### PR TITLE
Ugact 569 migrate to GitHub actions

### PR DESCRIPTION
**Pull request by KamilChoroba**
* Use github actions to build + publish (release, gh-pages) files to release and gh-pages branch.
* This will trigger jenkins jobs "BuildProd" and "DeployProd"
* Skip release + gh-pages jobs if change did not happen on master branch
* Store artifacts to share between jobs
* Remove travis integration

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+  publish-release:
+    needs: build
+    if: ${{ github.ref == 'refs/heads/main' }}
+    runs-on: ubuntu-latest
+    steps:
+      - run: echo "Deploying to production server on branch $GITHUB_REF"
+      - name: Check out branch ${{ github.ref }}
+        uses: actions/checkout@v2
+        with:
+          path: main
```
Comment (mpuppin) [inline .github/workflows/main.yml:33]: "main" or "master"?

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+          path: release
+      - name: Read .nvmrc
+        working-directory: main
+        run: echo "##[set-output name=NVMRC;]$(cat .nvmrc)"
+        id: nvm
+      - name: Use nvm version
+        uses: actions/setup-node@v2
+        with:
+          node-version: "${{ steps.nvm.outputs.NVMRC }}"
+      - name: Build
```
Comment (mpuppin) [inline .github/workflows/main.yml:47]: why you have to build again? cannot use the artifact created in the build step?

Comment (lakhaNamdhari) [reply .github/workflows/main.yml:55]: I think this is what he mentioned in daily that he is working on about stashing artifacts.

File: .github/workflows/main.yml (hunk 1 above)
Comment (KamilChoroba) [reply .github/workflows/main.yml]: I've changed to master

File: .github/workflows/main.yml (hunk 2 above)
Comment (KamilChoroba) [reply .github/workflows/main.yml:55]: Sharing data between jobs is handled by https://docs.github.com/en/actions/guides/storing-workflow-data-as-artifacts.

This means files/directories/artifacts are uploaded to github, which will require storage on github.

Github itself provides a free usage of "used minutes" and "used storage": https://docs.github.com/en/billing/managing-billing-for-github-actions/viewing-your-github-actions-usage#viewing-github-actions-usage-for-your-organization.

But I don't know what these limits are. Checking with application-platform.

Maybe its not worth to share data between jobs

Comment (KamilChoroba) [reply .github/workflows/main.yml]: Storing artifacts is now in place

**Review by lakhaNamdhari — APPROVED** [review]
Very nice Kamil :)

**Pull request merged**